sirius-valley/lerni-mobile
# [LER-BUG-31] Trivia choices not appearing

On this PR: 
- Added new library with a new scrollview. This component has the capability to style the scrollbar on android and ios. Now when questions are to long a scrollbar appears. The design is approved by Fabri. 

To test this feature: 
- You can go to /components/trivia/AnswerButtons/index.tsx and paste this two props on AnswerButton component to make answers very long in any trivia: 
` 
answer={
"asdasdadasd asdasdadasd asdasdadasd asdasdadasd asdasdadasd asdasdadasd asdasdadasd asdasdadasdhandleAnswerhandleAnswerhandleAnswer"
}
onPress={() => handleAnswer(option)}
`
- Then play any trivia, you can also test it on small phones like iphone SE. You should see the scrollbar rendered.
